Upstream-Status: Pending

This patch fixes following issue with automake 1.12

| configure.in:193: error: automatic de-ANSI-fication support has been removed

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/05/03

Index: minicom-2.6.1/configure.in
===================================================================
--- minicom-2.6.1.orig/configure.in
+++ minicom-2.6.1/configure.in
@@ -190,7 +190,6 @@ AC_CHECK_HEADERS(stdarg.h varargs.h term
 
 dnl Checks for typedefs, structures, and compiler characteristics.
 AC_C_CONST
-AM_C_PROTOTYPES
 AC_TYPE_UID_T
 AC_TYPE_MODE_T
 AC_TYPE_PID_T
